src/opnsense/mvc/app/models/OPNsense/Backup/NextcloudSettings.xml

```diff
@@ -52,5 +52,11 @@
             <default>OPNsense-Backup</default>
             <ValidationMessage>The Backup Directory can only consist of alphanumeric characters.</ValidationMessage>
         </backupdir>
+        <dateformat type="TextField">
+            <Required>Y</Required>
+            <mask>/[dDjlNSwzWFmMntLoYyaABgGhHisuveIOPTZcrU\.\-\_\:\(\)]+/</mask>
+            <default>Y-m-d_h:m:s</default>
```

Comment:
The i for minutes must stay, otherwise you break the format. Also in the frontend.

Reply:
Nice catch